src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/SearchPageObject.java

```diff
@@ -15,10 +15,14 @@
 
   protected static final String PAGINATION_PAGES_CSS = ".paginator-page";
 
-  @FindBy(css = "#searchInput")
+  @FindBy(css = ".wds-global-navigation__search-toggle")
+  protected WebElement searchButton;
+  @FindBy(css = ".wds-global-navigation__search-input-wrapper input")
   protected WebElement searchInput;
+  @FindBy(css = ".wds-global-navigation__search-submit")
+  protected WebElement searchSubmit;;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)